tests/system/CodeIgniterTest.php

```diff
@@ -96,6 +96,7 @@ public function testRunClosureRoute()
         $_SERVER['argc'] = 2;
 
         $_SERVER['REQUEST_URI'] = '/pages/about';
+        $_SERVER['SCRIPT_NAME'] = '/index.php';
```

Comment:
Why is this now required? Will this break similar tests for developers?

Reply:
See #7643
The constructor of `IncomingRequest` changes the URI object.
So when you specify `REQUEST_URI`, it is better to set `SCRIPT_NAME`.

If the class is changed so that it does not depend on the superglobal, 
there is no need to set the superglobal like that. 
However, I am not sure when it can be fixed. #7282 is too big.